### PR TITLE
12.0 pos multi session port fixes from 10

### PR DESCRIPTION
**Pull request by em230418**
Ported this using cherry-pick. https://github.com/it-projects-llc/pos-addons/pull/1052

### CODE REVIEW

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
Tour pos_order_cancel_tour failed at step .tables .table

```
************* Module pos_multi_session
Odoo version in link https://www.odoo.com/apps/modules/10.0/pos_multi_session/ is not valid
pos_multi_session/__init__.py:1: [E7989(readme-versions), ] Wrong version in module's README.rst
```

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Even if a floor plan is set in a multi-session configuration, there is not floor plan on the POS.

File: pos_multi_session/static/src/js/pos_multi_session.js [hunk 1]
```diff
@@ -17,9 +17,8 @@ odoo.define('pos_multi_session', function(require){
     var chrome = require('point_of_sale.chrome');
     var longpolling = require('pos_longpolling.connection');
     var rpc = require('web.rpc');
-    var PosBaseWidget = require('point_of_sale.BaseWidget');
+    var Model = require('web.Model');
```
Comment (KolushovAlexandr) [inline pos_multi_session/static/src/js/pos_multi_session.js]: it is obsolete since odoo 11.0. rpc is used now

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
Odoo version in link http://runbot.it-projects.info/demo/pos-addons/10.0 is not valid
pos_order_cancel_restaurant/__init__.py:1: [E7989(readme-versions), ] Wrong version in module's README.rst

File: pos_multi_session/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`4.2.10`
+--------
+
+- **Fix:** Synchronization error after an order was created offline
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_multi_session/doc/changelog.rst:5]: :+1: checked

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by KolushovAlexandr — APPROVED** [review]
I approve code changes

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**